tests/python/requirements/PySafetyBearWithoutMockTest.py

```diff
@@ -0,0 +1,25 @@
+from queue import Queue
+from bears.python.requirements.PySafetyBear import PySafetyBear
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+
+
+class PySafetyBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = PySafetyBear(Section('name'), Queue())
+
+    def test_without_vulnerability(self):
+        self.check_validity(self.uut, ['# whee', 'foo==1.0', '# whee'])
+
+    def test_with_vulnerability(self):
+        self.check_invalidity(self.uut, ['bottle<0.12.10', 'bottle==0.10.1'])
+
+    def test_with_cve_vulnerability(self):
+        self.check_invalidity(self.uut, ['bottle==0.1.10', '-e .'])
```

Comment:
which of `'bottle==0.1.10'` and `'-e .'` causes the vulnerability?
why have two?

Reply:
Thats are two typo  here :disappointed: first only testing using `bottle==0.10.1` not any `bottle=0.1.10` and second  in heading of commit its `of` not `to`